SettingsList.py

```diff
@@ -2519,7 +2530,9 @@ def __init__(self, name, gui_text, min, max, default, step=1,
             'remove':    'Remove (Keysy)',
             'vanilla':   'Vanilla Locations',
             'dungeon':   'Dungeon Only',
-            'keysanity': 'Anywhere (Keysanity)'
+            'overworld': 'Overworld Only (Keysanity)',
```

Comment:
I don't think it makes much sense to say "(Keysanity)" after all of these. I could see that leading to confusion if people say "I'm playing Keysanity." Maybe keep to only the last one, or remove them all? Same for the others.

Reply:
Done.